src/mudlet.cpp

```diff
@@ -259,6 +260,25 @@ mudlet::mudlet()
     scanForQtTranslations(getMudletPath(qtTranslationsPath));
     loadTranslators(mInterfaceLanguage);
 
+    // In the following the replacement parameter(s) are supplied at points of use:
+#if defined(Q_OS_WIN32)
```

Comment:
@vadi2 if we want to try something similar for the MacOs case we need to know what the name Qt uses for the MacOs style - if someone can try starting mudlet from a Mac command line with a `-style=Nonsense` argument then on Linux/FreeBSD and indeed Windows the Qt libraries will moan about an invalid style but to get a report about the available ones it can be done by inserting just above here:
```cpp
qDebug().nospace().noquote() << "mudlet::mudlet() INFO - styles available: \n" << QStyleFactory::keys().join(QChar::LineFeed);
```
On my main PC in linux using the system Qt libs (and the Kvantum package installed) I get:
```sh
stephen@Ripley:~/src/mudlet-dev/Mudlet/mudlet/build-mudlet-Desktop_Qt_5_x_GCC8_System-Debug$ ./mudlet -style=Nonsense
QApplication: invalid style override passed, ignoring it.

...
mudlet::mudlet() INFO - styles available: 
Breeze
bb10dark
bb10bright
kvantum-dark
kvantum
cleanlooks
gtk2
cde
motif
plastique
Oxygen
Windows
Fusion

...
```
So if we can do this on a MacOs platform and check what style(s) cause the problem we can try the same sort of fix for JUST that style...

Reply:
Ah, since I left something behind and had to do another commit I've added that line to the code as well - so we need to see what styles show the problem on Macs...